data/test_contacts.json

```diff
@@ -0,0 +1,4 @@
+[
```

Comment:
Where are you saving project data to? Having a `data/` directory inside a project is a bit dangerous because you might commit sensitive data by accident.

I had this file in run_scripts

Reply:
Thanks for pointing out this @as2388. @lukechurch suggested to remove it from run_scripts as it is a a data file. For other sensitive project outputs/input data I will store them in a diff directory outside of the project directory.

Reply:
Can we call this folder `configuration` then?

(@lukechurch I assume this will move into the configuration json in the near future so no need to spend too much time thinking about this)